pallets/tinlake-investor-pool/src/lib.rs

```diff
@@ -300,10 +316,23 @@ pub mod pallet {
 	pub enum Event<T: Config> {
 		/// Pool Created. [pool, who]
 		PoolCreated(T::PoolId, T::AccountId),
+		/// Pool metadata updated. [pool, metadata]
+		PoolMetadataSet(T::PoolId, Vec<u8>), // Vec<u8, T::PoolMetadataLimit>
 		/// Epoch executed [pool, epoch]
 		EpochExecuted(T::PoolId, T::EpochId),
 		/// Epoch closed [pool, epoch]
 		EpochClosed(T::PoolId, T::EpochId),
+		/// Fulfilled orders collected [pool, tranche, end_epoch, user, payout_currency_amount, payout_token_amount, remaining_supply_currency, remaining_redeem_token]
+		OrdersCollected(
+			T::PoolId,
+			T::TrancheId,
+			T::EpochId,
+			T::AccountId,
+			T::Balance,
+			T::Balance,
+			T::Balance,
+			T::Balance,
+		),
```

Comment:
It seems that we could make use of the `OutstandingCollections` type here to simplify and make it neater.
 
```suggestion
		/// Fulfilled orders collected [pool, tranche, end_epoch, user, outstanding_collections]
		OrdersCollected(
			T::PoolId,
			T::TrancheId,
			T::EpochId,
			T::AccountId,
			OutstandingCollections<T::Balance>
		),
```

Reply:
That's neat! Didn't know that was possible.